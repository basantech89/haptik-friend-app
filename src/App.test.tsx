import { cleanup } from '@testing-library/react'

afterEach(cleanup)

it('should take a snapshot', () => {
	// const { getByText } = render(<App />)
	// expect(getByText(/home container/i)).toBeInTheDocument()
})
