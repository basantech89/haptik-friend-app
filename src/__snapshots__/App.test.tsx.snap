// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation0"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular AppHeader-toolbar-1 MuiToolbar-gutters"
    >
      <a
        aria-current="page"
        class="AppHeader-title-2"
        href="/"
      >
        HAPTIK
      </a>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-text AppHeader-themeToggleButton-3"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          <svg
            height="512pt"
            viewBox="-11 0 512 512.002"
            width="512pt"
          >
            <path
              d="M490 267.004c0 135.308-109.688 245-245 245-135.309 0-245-109.692-245-245 0-135.313 109.691-245 245-245 135.313 0 245 109.687 245 245zm0 0"
              fill="#1a2b63"
            />
            <path
              d="M367.648 304.113h5.457c18.278-20.355 29.622-47.054 30.282-76.343-10.63 4.246-18.399 14.128-19.61 25.953h-16.129c-13.918 0-25.195 11.285-25.195 25.199-.004 13.914 11.277 25.191 25.195 25.191zm0 0M480.148 198.055a50.11 50.11 0 00-16.96 3.105l5.23 5.23c.375.376.582.88.582 1.415 0 .539-.207 1.039-.582 1.414l-9.2 9.199a1.981 1.981 0 01-1.413.582c-.535 0-1.04-.207-1.414-.582l-8.368-8.367a50.364 50.364 0 00-14.75 21.32 31.26 31.26 0 00-16.71-5.777 176.41 176.41 0 014.437 39.41c0 13.437-1.52 26.527-4.387 39.11h70.594a246.798 246.798 0 002.793-37.11c0-23.945-3.45-47.078-9.852-68.95zm0 0"
              fill="#2e4c89"
            />
            <path
              d="M270 208.922c0-13.914-11.277-25.195-25.195-25.195h-16.13c-1.62-15.836-15-28.192-31.265-28.192-3.265 0-6.41.512-9.37 1.438-13.583 19.277-21.571 42.765-21.571 68.086 0 3.046.113 6.066.34 9.058h77.996c13.918-.004 25.195-11.281 25.195-25.195zm0 0M134.504 128.121a50.47 50.47 0 00-2.672-.078 50.084 50.084 0 00-24.078 6.133c-9.176-16.348-25.434-28.18-44.617-31.332-32.309 35.77-54.203 81.117-60.934 131.27H71.72c7.562-42.552 30.476-79.86 62.785-105.993zm0 0"
              fill="#445ea0"
            />
            <path
              d="M309.828 174.344a2.46 2.46 0 01-2.398-1.918c-1.98-8.77-3.09-9.875-11.86-11.856a2.456 2.456 0 01-1.914-2.394c0-1.153.797-2.149 1.914-2.399 8.774-1.98 9.88-3.09 11.86-11.859a2.453 2.453 0 012.398-1.914 2.446 2.446 0 012.395 1.914c1.984 8.77 3.09 9.875 11.859 11.855a2.467 2.467 0 011.918 2.403 2.468 2.468 0 01-1.918 2.394c-8.77 1.98-9.879 3.09-11.86 11.856a2.453 2.453 0 01-2.394 1.918zm0 0"
              fill="#fff"
            />
            <path
              d="M242.828 38.34a2.917 2.917 0 01-2.844-2.27c-2.347-10.394-3.66-11.71-14.058-14.058a2.913 2.913 0 010-5.684c10.394-2.348 11.71-3.66 14.058-14.058A2.912 2.912 0 01242.828 0a2.908 2.908 0 012.84 2.27c2.352 10.398 3.664 11.71 14.059 14.058A2.918 2.918 0 01262 19.172a2.92 2.92 0 01-2.273 2.84c-10.395 2.347-11.711 3.664-14.059 14.054a2.908 2.908 0 01-2.84 2.274zm0 0"
              fill="#ffaf10"
            />
            <path
              d="M399.172 178.344a3.228 3.228 0 01-3.14-2.508c-2.594-11.48-4.044-12.934-15.524-15.527a3.215 3.215 0 010-6.273c11.48-2.595 12.93-4.048 15.523-15.528a3.218 3.218 0 013.14-2.504 3.207 3.207 0 013.138 2.504c2.597 11.48 4.046 12.933 15.527 15.523a3.231 3.231 0 012.512 3.145 3.225 3.225 0 01-2.512 3.133c-11.48 2.593-12.934 4.046-15.527 15.523a3.214 3.214 0 01-3.137 2.512zm0 0"
              fill="#ffdd8d"
            />
            <g
              fill="#ffd00d"
            >
              <path
                d="M355.172 83.004a3.981 3.981 0 01-3.879-3.102c-3.207-14.191-5-15.988-19.191-19.191A3.98 3.98 0 01329 56.832a3.976 3.976 0 013.102-3.883c14.191-3.203 15.984-5 19.191-19.191a3.973 3.973 0 013.879-3.102 3.967 3.967 0 013.879 3.102c3.21 14.191 5 15.984 19.195 19.191a3.983 3.983 0 013.106 3.883 3.993 3.993 0 01-3.106 3.879c-14.195 3.203-15.988 5-19.195 19.187a3.974 3.974 0 01-3.88 3.106zm0 0M342.715 327.316c18.394 10.746 30.785 30.692 30.785 53.485 0 2.054-.11 4.101-.313 6.133 30.235-31.77 48.813-74.72 48.813-121.93 0-17.723-2.617-35.23-7.777-52.04a6.28 6.28 0 00-5.989-4.429c-.39 0-.789.04-1.183.117a6.263 6.263 0 00-5.063 6.664c.293 3.575.434 6.766.434 9.743 0 43.793-24.086 82.058-59.707 102.257zm0 0M131.387 400.61h5.293l.093-.895c1.594-15.563 14.606-27.293 30.266-27.293a30.204 30.204 0 0117.652 5.652l1.079.77.445-1.246c7-19.54 25.644-32.668 46.406-32.668a49.326 49.326 0 0123.598 6.011l.867.473.484-.863a61.736 61.736 0 016.84-9.813c-55.023-9.738-96.941-57.902-96.941-115.68 0-64.777 52.695-117.476 117.476-117.476 2.977 0 6.164.145 9.743.434a6.267 6.267 0 006.664-5.063 6.264 6.264 0 00-4.313-7.172c-16.809-5.164-34.32-7.777-52.039-7.777-97.598 0-177 79.398-177 177 0 54.387 24.664 103.113 63.387 135.605zm0 0"
              />
            </g>
            <path
              d="M131.387 400.61h5.293l.093-.895a30.309 30.309 0 016.809-16.293C115.266 352.035 98 310.504 98 265.004c0-92.559 71.414-168.738 162.031-176.363-4.984-.418-10-.637-15.031-.637-97.598 0-177 79.398-177 177 0 54.387 24.664 103.113 63.387 135.605zm0 0"
              fill="#ffaf10"
            />
            <path
              d="M119.648 399.61h16.13c1.62-15.833 15-28.188 31.261-28.188 6.8 0 13.09 2.168 18.234 5.84 6.957-19.43 25.528-33.332 47.348-33.332a50.084 50.084 0 0124.078 6.133c10.774-19.196 31.313-32.172 54.89-32.172 34.743 0 62.911 28.168 62.911 62.91a62.857 62.857 0 01-2.863 18.812h5.172c13.914 0 25.195 11.282 25.195 25.196-.004 13.914-11.281 25.195-25.2 25.195H119.649c-13.918 0-25.195-11.281-25.195-25.195-.004-13.914 11.277-25.2 25.195-25.2zm0 0"
              fill="#445ea0"
            />
            <path
              d="M110 434.922c0-11.137 9.063-20.195 20.195-20.195h16.133a5.002 5.002 0 004.973-4.493c1.383-13.511 12.687-23.699 26.289-23.699a26.22 26.22 0 0115.328 4.91c.86.61 1.875.93 2.906.93.5 0 1.004-.074 1.492-.23a4.997 4.997 0 003.215-3.086c6.43-17.954 23.563-30.016 42.64-30.016 7.571 0 15.071 1.91 21.68 5.52a4.992 4.992 0 006.758-1.941c10.254-18.27 29.618-29.618 50.528-29.618 20.418 0 38.39 10.629 48.707 26.64-8.692-24.335-31.938-41.753-59.258-41.753-23.578 0-44.113 12.976-54.887 32.171a50.084 50.084 0 00-24.078-6.132c-21.82 0-40.39 13.902-47.348 33.332a31.275 31.275 0 00-18.234-5.84c-16.262 0-29.64 12.355-31.262 28.187h-16.132c-13.915 0-25.196 11.286-25.196 25.2 0 12.882 9.676 23.5 22.153 25.003-4.043-3.695-6.602-8.996-6.602-14.89zm0 0"
              fill="#2e4c89"
            />
            <path
              d="M146.316 81.52a3.013 3.013 0 01-4.242 0l-9.199-9.2a3.013 3.013 0 010-4.242l9.2-9.2a3.006 3.006 0 014.241 0l9.2 9.2a3.006 3.006 0 010 4.242zm0 0"
              fill="#ffaf10"
            />
            <path
              d="M53.316 203.129a3.013 3.013 0 01-4.242 0l-9.199-9.2a3.013 3.013 0 010-4.242l9.2-9.199a3.006 3.006 0 014.241 0l9.2 9.2a3.006 3.006 0 010 4.242zm0 0"
              fill="#fff"
            />
            <path
              d="M459.926 219.129a3.013 3.013 0 01-4.242 0l-9.2-9.2a3.013 3.013 0 010-4.242l9.2-9.199a3.006 3.006 0 014.242 0l9.199 9.2a3.006 3.006 0 010 4.242zm0 0"
              fill="#ffaf10"
            />
          </svg>
          <svg
            viewBox="0 0 512 512"
          >
            <path
              d="M255.23 414.029l88.641-222.402c3.709-9.306 14.26-13.844 23.566-10.135 9.306 3.709 13.844 14.26 10.135 23.566l-84.653 212.396-37.689-3.425z"
              fill="#90a4ae"
            />
            <path
              d="M290.171 220.482l.027-.063-.026.004-.001.059z"
              fill="#fff9c5"
            />
            <path
              d="M367.439 181.492c-9.306-3.709-19.857.828-23.566 10.135l-23.211 58.237a82.626 82.626 0 0117.587 4.868c5.791 2.308 11.193 5.181 16.121 8.545l23.204-58.218c3.708-9.308-.829-19.858-10.135-23.567z"
              opacity="0.1"
            />
            <path
              d="M399.536 100.961l-66.884 7.189-42.481 49.475.007 62.783.03-.012-.01.022c18.552-3.445 39.208-1.638 59.202 6.331 19.867 7.918 36.018 20.704 47.109 35.837l.194.354 34.942-34.596 3.898-86.328-36.007-41.055z"
              fill="#fff9c5"
            />
            <path
              d="M461.612 453.139c-24.34-38.907-106.776-67.539-204.726-67.539-97.947 0-180.38 28.63-204.723 67.535l13.675.004 29.677 19.032 31.205 3.392 21.992-12.706 34.908-12.732 18.733 5.087 39.853 20.351 26.286-4.24 33.917-12.655 24.59-7.695 31.373 7.695 25.438 16.895 40.701-5.087 37.101-17.337z"
              fill="#ffca28"
            />
            <path
              d="M427.368 469.141l34.245-16.002c-24.34-38.906-106.776-67.539-204.726-67.539-22.521 0-44.217 1.517-64.579 4.32 101.042 11.588 185.552 40.647 235.06 79.221z"
              opacity="0.1"
            />
            <path
              d="M508.696 457.316c-24.053-16.313-55.138-16.313-79.191 0-18.918 12.83-43.365 12.83-62.285 0-24.053-16.313-55.138-16.313-79.191 0-18.918 12.83-43.366 12.83-62.285 0-24.053-16.313-55.138-16.313-79.191 0-18.919 12.83-43.366 12.83-62.285 0-24.053-16.313-55.138-16.313-79.191 0a7.53 7.53 0 008.453 12.465c18.918-12.83 43.366-12.831 62.285 0 24.054 16.312 55.139 16.312 79.191 0 18.918-12.83 43.366-12.831 62.285 0 24.054 16.312 55.139 16.312 79.191 0 18.918-12.83 43.366-12.831 62.285 0 12.027 8.156 25.812 12.234 39.596 12.234 13.784 0 27.569-4.078 39.595-12.234 18.918-12.83 43.366-12.83 62.285 0a7.531 7.531 0 008.453-12.465z"
              fill="#1eb8e8"
            />
            <g
              fill="#ef495b"
            >
              <path
                d="M290.178 220.409c-11.098-15.259-27.334-28.156-47.327-36.125-20.003-7.973-40.671-9.778-59.228-6.327 0 0 90.046-127.161 215.914-76.995 0 0-66.007 17.655-109.365 119.521l.006-.074z"
              />
              <path
                d="M396.749 262.884c18.552-3.443 39.208-1.636 59.202 6.333 20.003 7.973 36.247 20.878 47.342 36.148 0 0 22.11-154.238-103.758-204.405 0 0 35.767 58.218-2.831 161.98l.045-.056z"
              />
            </g>
            <circle
              cx="85.695"
              cy="115.685"
              fill="#ffca28"
              r="41.025"
            />
            <g
              fill="#ff9800"
            >
              <path
                d="M85.696 60.193a7.53 7.53 0 01-7.53-7.53V37.515a7.53 7.53 0 0115.06 0v15.148a7.53 7.53 0 01-7.53 7.53zM130.255 78.651a7.503 7.503 0 01-5.324-2.206 7.53 7.53 0 010-10.65l10.711-10.711a7.53 7.53 0 0110.649 10.65L135.58 76.445a7.507 7.507 0 01-5.325 2.206zM163.86 123.211h-15.147a7.53 7.53 0 010-15.06h15.147a7.53 7.53 0 010 15.06zM140.966 178.482a7.5 7.5 0 01-5.324-2.206l-10.711-10.711a7.53 7.53 0 0110.649-10.65l10.711 10.711a7.53 7.53 0 01-5.325 12.856zM85.696 201.375a7.53 7.53 0 01-7.53-7.53v-15.148a7.53 7.53 0 0115.06 0v15.148a7.53 7.53 0 01-7.53 7.53zM30.424 178.482a7.503 7.503 0 01-5.324-2.206 7.53 7.53 0 010-10.65l10.71-10.711a7.53 7.53 0 0110.649 10.65l-10.71 10.711a7.507 7.507 0 01-5.325 2.206zM22.678 123.211H7.53a7.53 7.53 0 010-15.06h15.148a7.53 7.53 0 010 15.06zM41.135 78.651a7.503 7.503 0 01-5.324-2.206L25.1 65.735a7.53 7.53 0 0110.649-10.65l10.71 10.711a7.53 7.53 0 01-5.324 12.855z"
              />
            </g>
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </header>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      style="outline: none; display: none;"
      tabindex="-1"
    />
    <div>
      <div
        class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
        role="progressbar"
        style="width: 40px; height: 40px;"
      >
        <svg
          class="MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
